### DISCUSSION TITLE
Summary anchors don't seem to work

### DISCUSSION THREAD

**Discussion by xt0rted** [community/community#16863] [Actions]
Based on my understanding of the code if an anchor is emitted with the ID `r0` it's changed to `user-content-r0` to ensure it's unique. There's also some extra logic that should let you link to that anchor using `#r0`. This works on the [checks page](https://github.com/xt0rted/dotnet-run-script/pull/10/checks?check_run_id=6445629940) but not the [run summary](https://github.com/HubbellCorp/SweetBlue/actions/runs/2341111644) page.

Upon further inspection this seems to work if you step through the `user-content` behavior code, but not when it runs without stepping through it.

**Comment by edhends** [comment]
Hello. You can report the bug here: https://support.github.com/contact/bug-report
